Feature/profile

RojasVSantiago opened the pull request.

RojasVSantiago committed:
Drawer

Drawer file changed.
Now drawer show profile photo of google account.

RojasVSantiago committed:
profile data

Profile tab edited:
now shows you google profile pick, name and mail.

RojasVSantiago committed:
Text

Now in profile tab is showed the amount of cocoms ended.

RojasVSantiago committed:
Button

Button 'go to reports' is working.

RojasVSantiago committed:
logout button

Text displayed in logout button are now in spanish

SAngelCuadrosV commented on lib/screens/profile/profile_tab.dart:
```diff
@@ -1,80 +1,164 @@
+        _isLoading = false;
+      });
+    } catch (e) {
+      _count = null;
```
Si hay un error, no debería ser nulo ya que quedaría algo como 
`Cocoms completadas: `

Si hay error, lo mejor sería colocar algo como un mensaje en la variable, remplazar la variable `_count ` por algo como `_message`
Si hay error, colocar `Visita la ventana de Reportes para ver las Cocoms que has completado.`
Si no hay error, colocar `Cocoms completadas: cocmosCount`

SAngelCuadrosV commented on lib/widgets/profile_pick.dart:
```diff
@@ -0,0 +1,66 @@
+
+import '../service/auth/auth_service.dart';
+
+class ProfilePick extends StatefulWidget {
```
Cambiar `ProfilePick` por `CircularProfilePicture`

RojasVSantiago committed:
Message and Class

Added message when get an error counting in profile tab.
Changed file name and class name: profile pick -> circular profile pick

SAngelCuadrosV commented on lib/widgets/circular_profile_pick.dart:
```diff
@@ -2,14 +2,14 @@ import 'package:flutter/material.dart';
 
-class ProfilePick extends StatefulWidget {
-  const ProfilePick({super.key});
+class CircularProfilePick extends StatefulWidget {
```
`Picture`, `Pick` no tiene que ver con fotos.

RojasVSantiago committed:
Circular profile photo

fine name changed: circular profile pick -> circular profile photo.
changelog updated.
